package.json

```diff
@@ -76,18 +76,18 @@
     "@types/qs": "^6.9.7",
     "@types/sinon": "^10.0.6",
     "@types/stats-lite": "^2.2.0",
-    "@uniswap/default-token-list": "^4.1.0",
+    "@uniswap/default-token-list": "^11.2.0",
     "@uniswap/permit2-sdk": "^1.2.0",
     "@uniswap/router-sdk": "^1.3.0",
-    "@uniswap/sdk-core": "^3.0.1",
-    "@uniswap/smart-order-router": "3.13.2-hotfix",
-    "@uniswap/token-lists": "^1.0.0-beta.24",
-    "@uniswap/universal-router-sdk": "^1.3.0",
+    "@uniswap/sdk-core": "^3.2.6",
+    "@uniswap/smart-order-router": "^3.13.4",
```

Comment:
Let's change this `3.13.5`, and remove the caret at the beginning?

Reply:
done